--- /var/lib/gems/1.9.1/gems/qtbindings-4.6.3.1/lib/Qt/qtruby4.rb	2010-10-25 22:03:56.136190328 +0200
+++ /var/lib/gems/1.9.1/gems/qtbindings-4.6.3.1/lib/Qt/qtruby4.rb	2010-10-25 22:10:29.745867254 +0200
@@ -2774,7 +2774,7 @@
           # Multiple matches are an error; the equality test below _cannot_ be commented out.
           # If ambiguous matches occur the problem must be fixed be adjusting the relative
           # ranking of the arg types involved in checkarg().
-          elsif current_match == best_match && id.smoke == chosen.smoke
+          elsif current_match == best_match && chosen && id.smoke == chosen.smoke
             puts "multiple methods matching, this is an error" if debug_level >= DebugLevel::Minimal
             chosen = nil
           end
